docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-6-2-0_{context}"]
+= New features
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
+
+.Integration with JIRA
+A new _Jira_ entry has been added in the left sidebar, under the Control Menu. This feature enables the user (Administrator, Architect) to create new Jira instances and edit existing instances.
+
+.Report custom metrics on MTA usage back through OpenShift Telemetry
+MTA reports back through OpenShift Telemetry as to the applications that it processes:
+
+* Number of applications in Inventory
+* Number of Analysis Run (aggregated)
+* Number of Assessments Run
+
+.Migration Waves management
+The technical lead of the migration project can create and modify applications and information on the migration.
```

Comment:
@HagayVider1 
I would suggest this:
".Migration waves
A migration wave is a tool for grouping applications to be migrated on a given schedule. In addition, a migration wave enables the user to export a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave for tracking."